drivers/Aqara/aqara-presence-sensor/src/fp2/discovery_helper.lua

```diff
@@ -33,10 +33,17 @@ function discovery_helper.get_device_create_msg(driver, device_dni, device_ip)
     return nil
   end
 
+  local device_label = device_info.label or "Aqara-FP2"
+  if device_dni then
+    -- To make it easier to distinguish devices, add the last four letters of dni to the label
+    -- for example, if device_info.label is "Aqara-FP2" and device_dni is "00:11:22:33:44:55", then device_label will be "Aqara-FP2 (4455)"
+    device_label = string.format("%s (%s)", device_label, string.sub(string.gsub(tostring(device_dni), ":", ""), -4))
```

Comment:
Can `device_info.label` ever be a user-set value? I think it makes sense to add the four digits in the case that the device name is just `Aqara-FP2`. But if the user sets a label I think we should avoid appending those numbers, since they'll show up on the device card in the mobile app.

Reply:
device_info.label is a value passed from firmware and is fixed to “Aqara-FP2” unless a derivative model of Aqara-FP2 appears. This is not a value that the user can set.